### DISCUSSION TITLE
Resource Suggestion

### DISCUSSION THREAD

**Discussion by gmax0** [rufuspollock/awesome-crypto-critique#84] [General]
I wonder if it'd be valuable to have a section pointing out some of the risks on a technical/market microstructure level to traders and consumers. Perhaps this might be too far down the rabbit hole.

Examples that come to mind (Ethereum DeFi risks)
https://arxiv.org/abs/2101.05511
https://arxiv.org/pdf/2009.14021.pdf
https://arxiv.org/pdf/2009.14021.pdf
https://www.coindesk.com/tech/2020/12/29/new-research-sheds-light-on-the-front-running-bots-in-ethereums-dark-forest/

https://www.singlelunch.com/2022/01/09/an-anatomy-of-bitcoin-price-manipulation/ (highly speculative, some interesting points nonetheless)

**Comment by rufuspollock** [comment]
@gmax0 I think these are great items to have e.g. in a section about risks and market manipulation at this level.
